resources/rebuild.sh

```diff
@@ -18,6 +18,19 @@ source "$GIT_ROOT_DIR/tools/functions"
 function install_dependencies {
     apt update
     apt install -y bc flex bison gcc make libelf-dev libssl-dev squashfs-tools busybox-static tree cpio curl patch docker.io
+
+    # Install Go
+    case $ARCH in
+        x86_64) arch=amd64 ;;
+        aarch64) arch=arm64 ;;
+    esac
```

Comment:
nit: you're only using this variable once, so could just inline it, and instead have a `case` statement when defining `$version`. Would avoid having two variables named `$arch` that are slightly different.

Reply:
indeed. fixed!